lib/saml/url_service.rb

```diff
@@ -82,22 +82,43 @@ def mhv_url
       build_sso_url(build_authn_context('myhealthevet'))
     end
 
+    def mhv_verified_url
```

Comment:
I wonder if this is a good opportunity to reduce the number of these functions, we could pass `type` and `authn_context` from `sessions_controller` and reduce a bunch of these to a single function